templates/Submissions/view.php

```diff
@@ -279,44 +279,7 @@
                     ?>
                 </li>
                 <?php } ?>
-                <li>
-                    <?php
-                    echo $this->Html->link(
```

Comment:
This buttom should still be there but rather redirect to the GitHub directory that contains the submission in the submission-data repository. Could you update this please?

Reply:
There are 3 buttons here. 1. Browse in GitHub, 2. Browse Files, 3. Download ZIP. I have removed only 2 and 3. We still have 1. Browse in GitHub in line 272 above which points to  the submission in the submission-data repository. Attached before and after screenshot here.  

![Screenshot 2023-08-03 at 10 48 35 AM](https://github.com/IO500/webpage/assets/3243744/de478b92-a5bf-4c66-aeb1-8cba84d14a88)

Reply:
Oh, you're right! Looks good then, thanks